doc/source/remote-instance.rst

```diff
@@ -0,0 +1,53 @@
+.. _pmm.amazon-rds-agentless:
+
+Adding Remote DB instance to |pmm|
+********************************************************************************
+
+There is a quick method for users to add DBaaS instances to |pmm| without having
+to hook into the Cloud Provider's API, and with no need to have |pmm-client|
+installed or any exporters running on the monitored node. The drawback of this
+approach is that you will not have visibility of host-level metrics (CPU,
+memory, and disk activity will not be captured nor displayed in |pmm|). 
+
+.. note: There is an alternative and more complex approach available for
+         |mysql-server|, which involves API-aware addition of an Amazon RDS /
+         Aurora DB instance.
+
+Both methods can be accessed in the |metrics-monitor| navigation menu by
+selecting the |pmm-add-instance| item in a |gui.pmm-dropdown| group:
+
+.. figure:: .res/graphics/png/metrics-monitor.cloud-menu.png
+
+Two database servers are currently supported by this method: |postgresql| and
+|mysql|.
+
+.. figure:: .res/graphics/png/metrics-monitor.add-rds-or-remote-instance.png
+
+
+Adding a Remote PostgreSQL Instance
+================================================================================
+
+To add a remote PostgreSQL DB instance, user should fill in at least the
+hostname, username and password field, and optionally also non-default hostname
+and port ones:
+
+.. figure:: .res/graphics/png/metrics-monitor.add-remote-postgres-instance.png
+
+
+Adding a Remote MySQL Instance
+================================================================================
+
+To add a remote MySQL DB instance, user should fill in at least the
+hostname, username and password field, and optionally also non-default hostname
+and port ones:
+
+.. figure:: .res/graphics/png/metrics-monitor.add-remote-mysql-instance.png
+
+Viewing Remote MySQL and PostgreSQL Instances
+================================================================================
+
```

Comment:
Add a section that shows how to navigate to the Remote instances screen. Use this image:
![image](https://user-images.githubusercontent.com/4398871/47671221-a1150a00-db74-11e8-84ab-3985588d1cdb.png)
show the highlight around PMM submenu, and around the RDS and Remote Instances link